src/time-picker/TimePicker.js

```diff
@@ -0,0 +1,226 @@
+import React from 'react';
+import { props, t, skinnable } from '../utils';
+import cx from 'classnames';
+import Dropdown from '../dropdown/Dropdown';
+import range from 'lodash/utility/range';
+import flatten from 'lodash/array/flatten';
+
+const options = [];
+
+export const H24 = '24h';
+export const H12 = '12h';
+export const inputError = {};
+const symbolRegex = /[\s,\-\+\.\|\/\\]/;
+const numberRegex = /^\d+$/;
+const separator = ':';
+const interval = 30;
+
+const pad = (num) => num <= 9 ? `0${num}` : num;
+const hoursInH12 = hour => hour + 12 !== 24 ? hour + 12 : 0;
```

Comment:
this function does not return a range of H12 values (`[1, 12]`). maybe find a better name?